code/drasil-code/package.yaml

```diff
@@ -2,7 +2,7 @@ name:                drasil-code
 version:             0.1.9.0
 author:              "Dan Szymczak, Steven Palmer, Jacques Carette, Spencer Smith"
 maintainer:          "Jacques Carette"
-synopsis:	     A framework for code and document generation for scientific software - Code SubPackage
+synopsis:	           A framework for code and document generation for scientific software - Code SubPackage
```

Comment:
I don't know if this is even an issue, but what's the reason for this extra space?

Reply:
So should I revert that change as the spacing may vary on each person's machine then?

Reply:
Definitely never use tabs.